src/pkg/services/m365/sites_test.go

```diff
@@ -81,10 +82,7 @@ func (suite *siteIntegrationSuite) TestSites_GetByID() {
 			assert.NotEmpty(t, site.WebURL)
 			assert.NotEmpty(t, site.ID)
 			assert.NotEmpty(t, site.DisplayName)
-			if site.OwnerType != SiteOwnerUnknown {
-				assert.NotEmpty(t, site.OwnerID)
-				assert.NotEmpty(t, site.OwnerType)
-			}
```

Comment:
we can't actually guarantee this is true.  We have separate test with well known groups for ID and email.